### DISCUSSION TITLE
Apakah Kita Mendukung Palestina ?!

### DISCUSSION THREAD

**Discussion by sooluh** [farizdotid/DAFTAR-API-LOKAL-INDONESIA#335] [Q&A] [answered]
Bismillahirrahmanirrahim, Assalamua'alaikum warahmatullahi wabarakatuh, dan salam sejahtera untuk kita semua.

Saat ini, kita semua disaksikan oleh konflik yang mengguncang Palestina, khususnya di Jalur Gaza. Tidak peduli agama, etnis, atau latar belakang apapun, kita harus bertanya pada diri kita sendiri: Apakah kita mendukung Palestina?

Kami ingin menggarisbawahi bahwa dukungan terhadap Palestina bukan hanya menjadi hak atau kewajiban bagi orang Muslim, tetapi juga merupakan tindakan kemanusiaan yang melampaui batas-batas agama dan etnis. Konflik ini telah berlangsung selama bertahun-tahun, dan dampaknya pada penduduk Palestina sangat merusak.

### Informasi Real-time

Untuk mendapatkan berita real-time tentang situasi di Palestina, kami ingin mengajak kalian untuk mengikuti channel Telegram yang menyediakan informasi terkini dalam bahasa Indonesia dan bahasa Inggris/Arab:

- [t.me/Save_Palestin](https://t.me/Save_Palestin) - Bahasa Indonesia
- [t.me/frontlinemedia](https://t.me/frontlinemedia) - Bahasa Indonesia
- [t.me/eyeonpal](https://t.me/eyeonpal) - Bahasa Inggris dan Arab

Namun, sekedar mengikuti berita belum cukup. Saya mengajak teman-teman semua untuk juga memahami sejarah dan akar konflik ini. Belajar tentang sejarah Palestina dan Israel akan membantu kita memahami situasi ini. Dengan pemahaman yang baik, kita dapat membantu memperkuat suara keadilan dan perdamaian di seluruh dunia.

### Palestina Dalam Angka (2 hari yang lalu)

Saat ini, pada bulan Oktober 2023, kita bersaksi lagi terjadinya konflik antara Zionis dan Palestina di Jalur Gaza. Zionis Israel terus melakukan tindakan genosida terhadap warga dan rakyat Palestina, menghancurkan pemukiman warga, bahkan merusak rumah sakit yang merupakan tempat manusia, termasuk dokter, perawat, dan anak-anak.

- Meninggal dunia: Setidaknya 2750 orang
- Terluka: ± 9700 orang
- Hilang dalam reruntuhan: ±1000 orang
- Pengungsi: 1 juta orang lebih
- Bayi yang terbunuh: Lebih dari 800 bayi

### Pertanyaanya

Pertanyaan yang harus kita jawab adalah, "Apakah kita mendukung Palestina?" Jika jawaban Anda adalah "iya", kami akan dengan bangga memasang banner dukungan Palestina pada README.md repositori proyek ini. Mari bersatu untuk mengakhiri penderitaan yang tidak adil ini dan berdiri bersama sebagai suara kemanusiaan.

Ingatlah, mendukung Palestina adalah tentang membela nilai-nilai kemanusiaan, perdamaian, dan hak asasi manusia. Mari bersatu dalam usaha untuk menciptakan dunia yang lebih baik untuk semua orang, tanpa memandang latar belakang agama atau etnisitas. Terima kasih telah berpartisipasi dalam diskusi ini, dan mari kita terus berdoa untuk perdamaian di Palestina.

**Comment by sooluh** [comment]
Need a response from you,

@farizdotid @wanrabbae @bachors @dyazincahya @lakuapik @azharimm @granitebps @rzkytmgr @renomureza @penggguna @ibnux @feri-irawan @fandilladp @damasdev @abiamarulloh @nasrul21 @dotprima etc

Sorry for the mention.

**Comment by farizdotid** [comment, marked as answer]
@sooluh Waalaikumsalam, Boleh silahkan dilanjut kang untuk pemasangan banner Palestina.

**Comment by ibnux** [comment]
Waalaikumsalam

Dibuat template markdown yang bisa dipakai setiap orang

**Reply by sooluh** [reply]
Kebetulan saya sudah nemu ini pak, tapi masih kendala di "tautan harusnya kemana?"

https://github.com/Safouene1/support-palestine-banner

Saya sedang konsultasi dengan yang lebih paham terkait konflik ini, untuk dibuatkan website yg menampilkan informasi terkait Palestina ini.

**Reply by ibnux** [reply]
Sip

Tautan tidak wajib.

Sama seperti Ukraine. Cukup support aja

Dunia sudah tahu

Atau link telegram diatas dimasukkan ke linktree gitu

**Reply by sooluh** [reply]
Oh iya betul pak, sepertinya satu-satunya quick way untuk saat ini hanya itu.

**Comment by mininxd** [comment]
oh iya, saran nih
bannernya kalau bisa dikasih redirrect ke sebuah website, yaitu kesini https://s.id/standwithpalestine

biar ada yg berniatan untuk donasi

**Reply by sooluh** [reply]
Alhamdulillah sudah diarahkan ke website tsb

**Comment by gensart-ai** [comment]
Buat API buat data rincian di Palestina kang.. Data - data open yang penting buat publik tau.

**Reply by sooluh** [reply]
Untuk saat ini saya belum menemukan, kira-kira data apa yang pas untuk API-nya, pun untuk waktu senggang-nya juga belum ada, barangkali dari temen-temen ada yang mau bantu buatkan, bisa coba kolaborasikan atau gunakan dataset dari @TechForPalestine atau [https://techforpalestine.org](techforpalestine.org)

**Reply by mininxd** [reply]
bagaimana kalau API web kitabisa?
untuk membuat program status donasi

**Reply by sooluh** [reply]
On-progress API untuk data apa-apa yang harus di boycott https://discord.com/channels/1186702814341234740/1201643999073665064

**Reply by nasrul21** [reply]
> On-progress API untuk data apa-apa yang harus di boycott https://discord.com/channels/1186702814341234740/1201643999073665064

mau dong bang @sooluh  ikutan kontribusi, boleh join discord nya?

**Reply by sooluh** [reply]
silahkan mas @nasrul21 itu project dari @TechForPalestine, saya juga sebagai kontributor hehe
